### ISSUE TITLE
prebuild CI

### ISSUE THREAD

**Issue by mikolalysenko** [stackgl/headless-gl#23]
Need to run the prebuild scripts and publish to npm whenever a new version is released.  Not sure the best way to automate this yet.  At minimum, we need to support the following platforms:

* OS X (64-bit)
* Linux (32 and 64-bit)
* Windows 8 (32 and 64 bit)
* Windows 10 (32 and 64 bit)

**Comment by mikolalysenko** [comment]
Some options:

Linux: Use travis
OS X: Can use travis if you send them an email, otherwise this isn't so hard to do locally
Windows:  http://www.appveyor.com/ 

Possibly cool hooks:

https://github.com/mapbox/mapbox-gl-native/blob/a6ef907d5e647dec9ecee44f14ee5fdf2d44af06/scripts/node/after_script.sh#L13-L37

**Comment by springmeyer** [comment]
@mikolalysenko after you added me as a contributor (thanks!) I was able to enable an appveyor CI hook: https://ci.appveyor.com/project/Mapbox/headless-gl/history. However the one build that has run I triggered manually. My first testing commits to the `appveyor` branch appear not to be triggering new runs on appveyor. I think this might be because I need to be an admin. I'll be working on figuring this out in the next days.

Context: I'm aware that windows support is not fully working yet as per https://github.com/stackgl/headless-gl/issues/25. Nevertheless my motivation behind helping with Windows CI is to have it in place to help anyone who has the ability and time to push forward windows support. And once working keep things from regressing.
